torch/_dynamo/variables/builtin.py

```diff
@@ -1101,6 +1107,21 @@ def call_getattr(
                 unimplemented("tensor grad")
             else:
                 unimplemented("tensor grad")
+        elif name == "__bases__":
+            value = obj.as_python_constant()
+            if isinstance(value, type):
```

Comment:
You can't do it this way, this will cause the rest of the elif sled to get bypassed